apps/condo/domains/ticket/hooks/useClientCardTicketTableColumns.tsx

```diff
@@ -8,102 +10,149 @@ import { getFilterIcon } from '@condo/domains/common/components/Table/Filters'
 import { getDateRender, getTableCellRenderer } from '@condo/domains/common/components/Table/Renders'
 import { getFilteredValue } from '@condo/domains/common/utils/helpers'
 import { getSorterMap, parseQuery } from '@condo/domains/common/utils/tables.utils'
-import { TicketComment } from '@condo/domains/ticket/utils/clientSchema'
 import {
     getClassifierRender,
     getStatusRender,
-    getTicketDetailsRender,
+    getTicketDetailsRender, getTicketUserNameRender, getUnitRender,
 } from '@condo/domains/ticket/utils/clientSchema/Renders'
 import { IFilters } from '@condo/domains/ticket/utils/helpers'
 
-
 const renderCell = getTableCellRenderer()
 const renderTicketDetails = getTicketDetailsRender()
 
-export function useClientCardTicketTableColumns (tickets) {
+const getLastComment =  (ticket: GetTicketsQuery['tickets'][number]) =>
+    ticket.lastCommentWithOrganizationTypeAt || ticket.lastResidentCommentAt || ticket.lastCommentWithResidentTypeAt
+
+export function useClientCardTicketTableColumns (tickets: GetTicketsQuery['tickets'], currentTableTab) {
     const intl = useIntl()
     const NumberMessage = intl.formatMessage({ id: 'ticketsTable.Number' })
     const DateMessage = intl.formatMessage({ id: 'Date' })
     const StatusMessage = intl.formatMessage({ id: 'Status' })
     const DescriptionMessage = intl.formatMessage({ id: 'Description' })
     const ClassifierTitle = intl.formatMessage({ id: 'Classifier' })
     const AddressMessage = intl.formatMessage({ id: 'field.Address' })
+    const UnitMessage = intl.formatMessage({ id: 'field.UnitName' })
     const LastCommentMessage = intl.formatMessage({ id: 'pages.condo.phone.table.lastComment' })
+    const ContactMessage = intl.formatMessage({ id: 'Contact' })
 
     const router = useRouter()
     const { filters, sorters } = parseQuery(router.query)
     const sorterMap = getSorterMap(sorters)
 
-    const ticketCommentsWhere = useMemo(() => tickets.map(ticket => ({
-        AND: [
-            { ticket: { id: ticket.id } },
-            { createdAt: ticket.lastCommentAt },
-        ],
-    })), [tickets])
-    const { objs: ticketComments } = TicketComment.useObjects({
-        where: {
-            OR: ticketCommentsWhere,
+    const hasComments = tickets?.map(ticket => getLastComment(ticket)).some(Boolean)
+
+    const ticketCommentsWhere: TicketCommentWhereInput[] = useMemo(() =>
+        tickets?.map(ticket => ({
+            AND: [
+                { ticket: { id: ticket.id } },
+                { createdAt: getLastComment(ticket) },
+            ],
+        })),
+    [tickets])
+
+    const { data } = useGetPollTicketCommentsQuery({
```

Comment:
Don't use `getPollTicketComments` query here, it’s needed for polling on the ticket page. You need to use a different query here or write a new one